.jshintrc

```diff
@@ -10,7 +10,7 @@
     "latedef": "nofunc",
     "laxbreak": true,
     "maxdepth": 4,
-    "maxparams": 5,
+    "maxparams": 6,
```

Comment:
The ignoreline technique is nice because it localizes the effect to that line; but it is not great because it disables all lint rules. The header technique is nice because it localizes the effect to a specific rule; but it is not great because it covers over the entire file.

Is it possible to use the rule-specific -/+ to disable/enable the rule around a block of code? You'll have to find the warning/error number from the docs or running the linter and violating the rule.

See: http://stackoverflow.com/questions/12977661/is-there-a-way-to-suppress-jshint-warning-for-one-given-line